sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md

```diff
@@ -667,17 +667,14 @@ Accept wildcard characters: False
 
 ### -SharingDomainRestrictionMode
 
-Specifies the external sharing mode for domains.
+Specifies the sharing mode for external domains.
 
-The following values are:
+Possible values are:
+- None - Do not restrict sharing by domain
+- AllowList - Sharing is allowed only with external users that have account on domains specified within -SharingAllowedDomainList
+- BlockList - Sharing is allowed with external users in all domains expect in domains specified within -SharingBlockedDomainList
```

Comment:
Did you mean "except" here?

Reply:
Of course it should be "except" :) I just changed it in the branch, PR should be updated.